plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableDataStoreTests.php

```diff
@@ -2786,6 +2786,39 @@ public function test_order_data_saved_correctly_with_sync() {
 		$this->assertEquals( 'test_value', $r_order->get_meta( 'test_key', true ) );
 	}
 
+
+	/**
+	 * @testDox Test that perform_custom_order_validation throws a RouteException with a custom error.
+	 */
+	public function test_perform_custom_order_validation() {
```

Comment:
I suggest moving this test to `plugins/woocommerce/tests/php/src/Blocks/StoreApi/Routes/Checkout.php` it doesn't make much sense living here